packages/sdk-v2/test/proposeOptimisticRewards.ts

```diff
@@ -0,0 +1,102 @@
+import {
```

Comment:
This should be collocated next to its source file, not in a separate test/ directory structure.

Reply:
good call